src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -390,7 +392,18 @@ public Command getDriveToPoseCommand() {
   }
 
   /** Runs the drive at the desired speeds set in (@Link setGoalSpeeds) */
-  public void runVelocity() {
+  public void runVelocity(Rotation2d desiredRotation) {
+    if (isAngleAlign) {
+      ProfiledPIDController angleController =
+          new ProfiledPIDController(5, 0.0, 0.4, new TrapezoidProfile.Constraints(8.0, 20));
+      angleController.enableContinuousInput(-Math.PI, Math.PI);
+      double omega =
```

Comment:
Recommend logging current rotation, desired rotation, and computed omega to help debug in advantage scope